tests/unit/test_messages_emitter.py

```
@@ -524,6 +538,15 @@ def test_paused_resumed_error(get_initiated_emitter, tmp_path):
     ]
 
 
+def test_paused_cant_show(get_initiated_emitter, tmp_path):
+    """The Emitter cannot show messages when paused."""
+    emitter = get_initiated_emitter(EmitterMode.QUIET)
+
+    with emitter.pause():
+        with pytest.raises(RuntimeError):
```

Good, we need some kind of error instead of failing silently.